Makefile.am

```diff
@@ -115,7 +115,9 @@ po-pull:
 	TEMP_DIR=$$(mktemp --tmpdir -d anaconda-localization-XXXXXXXXXX) && \
 	git clone --depth 1 -b $(GIT_L10N_BRANCH) -- $(L10N_REPOSITORY) $$TEMP_DIR && \
 	cp $$TEMP_DIR/$(L10N_DIR)/*.po $(srcdir)/po/ && \
-	rm -rf $$TEMP_DIR
+	rm -rf $$TEMP_DIR && \
+	sed -i 's/X-Generator:\ Zanata 3\.5\.1//' $(srcdir)/po/kn.po && \
```

Comment:
We don't use Zanata so I guess we can drop this Hack?

Reply:
The problem is that is string exists on the po files, https://github.com/rhinstaller/anaconda-l10n/blob/master/master/kn.po#L35 and makes them unparseable by cockpit's translation mechanism.

Reply:
I think the correct resolution here is to change the po files and remove the Zanata header portion. I will make such a PR for the l10n repo.

Reply:
makes total sense, thanks!

Reply:
Removed this.